enzyme/Enzyme/Utils.cpp

```diff
@@ -632,26 +637,40 @@ Function *getOrInsertDifferentialFloatMemcpy(Module &M, Type *elementType,
   return F;
 }
 
-Function *getOrInsertMemcpyStridedBlas(Module &M, PointerType *T, Type *IT,
-                                       BlasInfo blas, bool julia_decl) {
+void callMemcpyStridedBlas(llvm::IRBuilder<> &B, llvm::Module &M, BlasInfo blas,
+                           llvm::ArrayRef<llvm::Value *> args,
+                           llvm::ArrayRef<llvm::OperandBundleDef> bundles) {
```

Comment:
We don't need the llvm namespace in a .cpp file where we have using namespace llvm

Reply:
We have 4 different using namespace llvm, one even globally, so the other three are not needed.
Let's clean that up separately.